AirbrakeLoggerConfigurationExtensions.cs

```diff
@@ -29,5 +29,19 @@ public static class AirbrakeLoggerConfigurationExtensions
         {
             return sinkConfiguration.Sink(new AirbrakeSink(config), restrictedToMinimumLevel);
         }
+
+        /// <summary>
+        /// Workaround to fix Airbrake call stack for Serilog.Sinks.Async
+        /// </summary>
+        /// <returns></returns>
+        public static LoggerConfiguration StackTraceWrapper(
+            this LoggerSinkConfiguration sinkConfiguration,
+            Action<LoggerSinkConfiguration> configureWrappedSink)
+        {
+            return LoggerSinkConfiguration.Wrap(
```

Comment:
nice way to do this! kudos
although this `w=>w.Async(x=>x.Airbrake(conf, LogEventLevel.Error))` looks terrifying :)

Reply:
any ideas to improve this?